File: app/src/main/java/com/charlie/sawl/widgets/meal/MealWidgetService.java
package com.charlie.sawl.widgets.meal;

import android.content.Intent;
import android.widget.RemoteViewsService;

public class MealWidgetService extends RemoteViewsService {

    @Override
    public RemoteViewsFactory onGetViewFactory(Intent intent) {
        return (new MealFactory(this.getApplicationContext(), intent));
    }
}

File: app/src/main/kotlin/com/charlie/sawl/widgets/meal/MealWidgetService.kt
package com.charlie.sawl.widgets.meal

import android.content.Intent
import android.widget.RemoteViewsService

class MealWidgetService : RemoteViewsService() {

    override fun onGetViewFactory(intent: Intent): RemoteViewsService.RemoteViewsFactory {
        return MealFactory(this.applicationContext)
    }
}